### DISCUSSION TITLE
CHOWN O QUE É? - CODAQUI.dev

### DISCUSSION THREAD

**Discussion by giscus** [codaqui/institucional#185] [Blog]
# CHOWN O QUE É? - CODAQUI.dev

Queremos democratizar o aprendizado tecnológico, e aproximar novas gerações do conteúdo tecnico.

https://www.codaqui.dev/blog/2024/03/13/chown-o-que-%C3%A9/

**Comment by endersonmenezes** [comment]
Parabéns pela postagem! Simples e direta!
